stl/src/xmath.hpp

```diff
@@ -7,14 +7,12 @@
 #include <cmath>
 #include <ymath.h>
 
-// MACROS FOR _Feraise ARGUMENT
 #define _FE_DIVBYZERO 0x04
 #define _FE_INEXACT   0x20
 #define _FE_INVALID   0x01
 #define _FE_OVERFLOW  0x08
 #define _FE_UNDERFLOW 0x10
 
-// FLOAT PROPERTIES

```

Comment:
Doesn't look like useless. Maybe just decapitalize?

Reply:
The ones in `yvals.h` could be de-capitalised?

https://github.com/microsoft/STL/blob/c2535c363805d8428790f3c2e2f2a139f5b90096/stl/inc/yvals.h#L34

https://github.com/microsoft/STL/blob/c2535c363805d8428790f3c2e2f2a139f5b90096/stl/inc/yvals.h#L310

https://github.com/microsoft/STL/blob/c2535c363805d8428790f3c2e2f2a139f5b90096/stl/inc/yvals.h#L316-L317

Reply:
I less care about those, these macros are clearer without comments than those I pointed out

Reply:
After looking at them, I dropped CURRENT DLL NAMES, MULTITHREAD PROPERTIES, and LOCK MACROS as being obvious/unnecessary, but lowercased INTEGER PROPERTIES because their names were kind of hard to understand otherwise.